### ISSUE TITLE
Convars or launch options to disable extra culling that would interfere with external software

### ISSUE THREAD

**Issue by Xenthio** [Facepunch/garrysmod-requests#2048]
The RTX Remix toolkit is coming out soon and it allows anybody to convert a fixed function pipeline game into a game with stunning raytracing and high quality asset replacements,

Garry's Mod has certain things that do not play nicely with RTX Remix, mostly small things such as culling of backfaces not visible to the camera (including some form of culling not present in other source engine games, which ruins the entire RTX effect)

they can be worked around using [a set of unofficial patches](https://github.com/BlueAmulet/SourceRTXTweaks) but that takes time,
(link has also more info about the kind of culling that would be nice to disable)

this feature request is to ask if we can have better methods of disabling these culling features that would benefit interms of ease of use and would help not upset any anti cheat by needing to modify more files than already has been modified with RTX Remix.

**Comment by ZH-Hristov** [comment]
Don't r_occlusion 0 and r_novis 1 fit the bill for what's needed for Remix?

**Comment by BlueAmulet** [comment]
They don't. Even with those set, I'm still experiencing objects disappearing when out of view and missing backfaces/hidden surfaces leaking light.

**Comment by Xenthio** [comment]
> Don't r_occlusion 0 and r_novis 1 fit the bill for what's needed for Remix?

not really, not only is culling still done but theres some odd gmod specific culling that cannot be disabled.

**Comment by sambow23** [comment]
Can we get an update on this? We're currently relying on binary patches that are hard to maintain between game updates and it would be nice if there was an official solution.
